prototypes/letter_clock/letter_clock.pde

```diff
@@ -0,0 +1,58 @@
+final int bitplane_n = 5;
+final int plane_size = 320*256;
+
+int[] background = new int[320*256 / 16 * bitplane_n];
+
+void setup() {
+  frameRate(60);
+  size(320, 256);
+  initOCS(bitplane_n);
+  
+  // Palette will probably need to be manually adjusted to fit your needs
+  for (int i = 0; i < 32; i++)
+    palette[i] = color(#000000);
+  palette[4] = color(#ffff00);
+  palette[9] = color(#ff00ff);
+  palette[11] = color(#ff00ff);
+  palette[12] = color(#ffff00);
+  palette[13] = color(#00ff00);
+  palette[18] = color(#ff00ff);
+  palette[19] = color(#ff00ff);
+  palette[20] = color(#ffff00);
+  palette[21] = color(#00ff00);
+    
+  // Experimental code:
+  String lines[] = loadStrings("background.txt");
```

Comment:
Can't you load the image directly and process it in a function? I don't want to keep in the repository intermediate conversion steps in obscure file format.

Reply:
`background.txt` is just a result of conversion with PNG2C with C-specific parts stripped. I wanted to use OCS for the prototype to be as "Amiga-like" as possible but AFAIK the image conversion code was not adapted for Processing.

Reply:
I guess you could easily write a function that reads in PNG image and produces an array of bitplanes and a palette. That could be of great value for other people using `ocs.pde` for prototyping!